config/initializers/job_configurations.rb

```diff
@@ -163,6 +163,10 @@
         cron: cron_24h,
         args: -> { [Time.zone.yesterday] },
       },
+      usps_auth_token_refresh: {
+        class: 'UspsAuthTokenRefreshJob',
+        cron: cron_5m,
```

Comment:
If you switch to renewing without the expiration check, then this could probably be once about every 12 or 13 minutes.

Reply:
Our reasoning for the 5 minutes was that if one job fails and it's near expiration, the other two in the 15 minute period should pick it up and hopefully finish it.

But, I can see that running the job less often is less Stuff going on, so there's a tradeoff there. We also have a failsafe built into the proofer methods, that also check the cache and then request a token if there isn't one.

With that in mind, would you prefer every 12 minutes?

Reply:
I'm OK with 5 minutes, but I have a slight preference for keeping the renewal & any retries towards the end of the window to reduce unnecessary traffic. I would say to focus mostly on the other parts of this PR.